### ISSUE TITLE
Need help with xmrig-proxy

### ISSUE THREAD

**Issue by FreeM1ne** [UnamSanctam/SilentCryptoMiner#2148]
I forgot the ports that the miners used to connect to, how can I find them out?

**Comment by UnamSanctam** [comment]
You can see it in the config.json file, up in the "bind" section.

**Comment by FreeM1ne** [comment]
The problem is that I replaced the config with a different config, and the ports all I do not remember them 5 pieces were

**Comment by UnamSanctam** [comment]
Do you not have the miner configuration that shows the ports? Or a "Remote Configuration" with them? Finding the incoming connections on a port that isn't listening can be difficult.

**Comment by FreeM1ne** [comment]
Maybe there is a possibility to run listening on all ports ?

**Comment by UnamSanctam** [comment]
Not really, do you not have your miner file left? Since then you can install it on a VM and then run the checker to see what port you're mining to.

**Comment by FreeM1ne** [comment]
I solved the problem by brute force. I wrote a script in python that created 15 folders with xmrig-proxy with 5000 ports in each folder and by running and checking the connections reassembled the ports again

**Comment by UnamSanctam** [comment]
Haha well done, I guess that's one solution.
